src/hrtf/mysofa.h

```diff
@@ -14,6 +14,16 @@ extern "C" {
 #include <stdint.h>
 #include <stddef.h>
 
+#ifdef _WIN32
+#ifndef MYSOFA_EXPORT
+    #define MYSOFA_EXPORT __declspec(dllexport)
+#else
+    #define MYSOFA_EXPORT __declspec(dllimport)
+#endif
+#else
+  #define MYSOFA_EXPORT
```

Comment:
so on `_WIN32` you can overide the `MYSOFA_EXPORT` by defining it prior to including the heaer file, but on other systems you cannot? sounds like inconsistent behaviour.

as mentioned otherwise, this should not be limited to `_WIN32` at all.